### DISCUSSION TITLE
How put a spacer between button and button?

### DISCUSSION THREAD

**Discussion by usuall** [zauberzeug/nicegui#1726] [Q&A] [answered]
### Question

Hello.
I want to know if I can put a spacer between button and button.
![2023-10-02 11 39 00](https://github.com/zauberzeug/nicegui/assets/277329/def850ea-ac0b-4867-860b-61ef09247cba)

**Comment by rodja** [comment, marked as answer]
You can use the `gap-` classes in the container:

```py
with ui.row().classes('gap-8'):
    ui.button('Button A')
    ui.button('Button B')
    ui.button('Button C')
```

But `ui.row()` already has a default gap of `2`. So it would be helpful what you have tried already and see a minimal example of your code.

**Reply by usuall** [reply]
oh. i got it.
